src/cljs/main/broadfcui/common/method/config_io.cljs

```diff
@@ -58,15 +58,15 @@
      (swap! state utils/deep-merge (:values props)))
    :render
    (fn [{:keys [props this]}]
-     (let [{:keys [default-hidden? entity-type? style]} props
+     (let [{:keys [default-hidden? entity-type? style edit-disabled?]} props
            id (gensym "io-table-")]
        [:div {:id id :style style}
         [Collapse {:title "Inputs"
                    :default-hidden? default-hidden?
                    :contents (this :-render-table :inputs)}]
         [Collapse {:style {:marginTop "1rem"}
                    :title "Outputs"
-                   :secondary-title (when (and entity-type? (seq (this :-get-defaultable-outputs)))
+                   :secondary-title (when (and entity-type? (seq (this :-get-defaultable-outputs)) (not edit-disabled?))
```

Comment:
Put this before `(seq (this :-get-defaultable-outputs))` to short circuit on easy computations